change/@microsoft-teams-js-4721f042-c277-4707-80ec-9a9dfe3ec273.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "minor",
+  "comment": {
```

Comment:
Can you please update this so the comment field looks like the following:

"comment": "Added `storage` capability with APIs to allows apps to discover if web storage is supported by the host client",

(please make sure to use backticks around "storage" in the comment string)